v2/anomalydetection/frontend-setup.mdx

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 2
+title: Frontend Setup
+hide_title: true
+---
+
+# Setting up the Frontend
+
+Setting up the frontend is relatively simple. You need to generate a request ID for each authentication event attempt. 
+This request ID is used as a parameter for the anomaly detection for detecting anomalies such as bot detection, impossible travel, and device information.
+
+Using it on other flows will not make a real difference, as, by their nature, other flows prevent this kind of anomalies.
+
+For detecting brute force attacks, password breaches, etc, you can skip these steps as the backend setup will handle this.
+
+### Generating a Request ID on the Frontend
+
+In order to generate a request ID, you need to import and initialise the SDK using your public API key. This SDK is used to generate a unique request ID for each authentication event attempt.
```

Comment:
We have not yet introduced what a public api key is. In fact, we have not introduced anything about API keys anywhere. So that needs to be clarified and communicated somewhere before.

Reply:
Done. added to the introduction page.